templates/legal/confidentiality-agreement.html

```diff
@@ -1,545 +1,626 @@
-                                            <option value="tfa_Utah" id="tfa_Utah" >Utah</option>
-                                            <option value="tfa_Vermont" id="tfa_Vermont" >Vermont</option>
-                                            <option value="tfa_Virginia" id="tfa_Virginia" >Virginia</option>
-                                            <option value="tfa_Washington" id="tfa_Washington" >Washington</option>
-                                            <option value="tfa_WestVirginia" id="tfa_WestVirginia" >West Virginia</option>
-                                            <option value="tfa_Wisconsin" id="tfa_Wisconsin" >Wisconsin</option>
-                                            <option value="tfa_Wyoming" id="tfa_Wyoming" >Wyoming</option>
-                                        </select>
+                      </li>
+                      <li class="p-list__item">
+                        <div id="tfa_7-D" class="oneField hintsBelow">
+                          <label id="tfa_7-L" for="tfa_7" class="label preField reqMark">
+                            Canonical contact <span>*</span>
+                          </label>
+
+                          <div class="inputWrapper">
+                            <input type="text"
+                                   id="tfa_7"
+                                   name="tfa_7"
+                                   value=""
+                                   placeholder=""
+                                   class="calc-CanContact required" />
+                            <span class="field-hint-inactive" id="tfa_7-H">
+                              <span id="tfa_7-HH" class="hint">Please add the Canonical account manager or contact</span>
+                            </span>
+                          </div>
                         </div>
-                      </div>
-                    </li>
-                    <li class="p-list__item">
-                      <div id="tfa_Address-D" class="oneField">
-                        <label id="tfa_Address-L" for="tfa_Address" class="label preField reqMark">Address <span>*</span></label>
-
-                        <div class="inputWrapper">
-                          <textarea cols="40" rows="4" id="tfa_Address" name="tfa_Address" class="required"></textarea>
+                      </li>
+                      <li class="p-list__item">
+                        <div class="htmlSection" id="tfa_5">
+                          <p>
+                            By clicking "I agree" below, I confirm that I have read and understood the terms of the confidentiality agreement, <a href="/legal/data-privacy">privacy policy</a> and <a href="/legal/data-privacy/unilateral-nda">privacy notice</a> and warrant that I have authority to enter into the <a href="https://assets.ubuntu.com/v1/c98a991d-Canonical%20NDA_Online%20Form%20%20-%2025th%20July%202024.docx.pdf">confidentiality agreement</a>
```

Comment:
```suggestion
                            By clicking "I agree" below, I confirm that I have read and understood the terms of the confidentiality agreement, <a href="/legal/data-privacy">privacy policy</a> and <a href="/legal/data-privacy/unilateral-nda">privacy notice</a> and warrant that I have authority to enter into the <a href="https://assets.ubuntu.com/v1/c98a991d-Canonical%20NDA_Online%20Form%20%20-%2025th%20July%202024.docx.pdf">confidentiality agreement</a>.
```
Minor thing, missing a full stop.